tools/generators/xcschemes/src/Generator/GeneratorArguments.swift

```diff
@@ -92,6 +92,12 @@ Path to a file that contains arguments for custom schemes.
         )
         private var targetAndExtensionHosts: [TargetID] = []
 
+        @Option(
+            parsing: .upToNextOption,
+            help: "List of exclude patterns to be applied on the scheme name."
+        )
+        var autogenerationSchemeNameExcludePatterns: [String] = []
+
```

Comment:
Can you move this to be under `autogenerationMode`? And then adjust all of the starlark code to pass it in the correct order?

Reply:
https://github.com/MobileNativeFoundation/rules_xcodeproj/pull/3027/commits/f0e091adb93265945ae1e0d155ba68a8b1ff6562

Let me know your thoughts! Happy to make further adjustments.